pkg/cluster/error.go

```diff
@@ -20,7 +20,7 @@ var (
 	errNoBackupExist     = errors.New("no backup exist for disaster recovery")
 	errInvalidMemberName = errors.New("the format of member's name is invalid")
 
-	errUnexpectedUnreadyMember = errors.New("unexpected unready member for selfhosted cluster")
+	errUnexpectedUnreadyMember = errors.New("unready member for selfhosted cluster")
```

Comment:
no. it is not. we wait the member to be added. when the member is added, we assume it will get started right away. also i do not feel this has anything to do with this pr.